Assets/Scripts/Player/PlayerBehaviour.cs

```diff
@@ -82,14 +88,46 @@ void Update()
     {
         // handoff update to current state
         currentState.UpdateState(this);
+        if (move.ReadValue<Vector2>() != Vector2.zero)
+        {
+            bool currentFacingRight = facingRight;
+            facingRight = move.ReadValue<Vector2>().x > 0;
+            if (currentFacingRight != facingRight) HandleFacing(); 
+        }
     }
 
-        void FixedUpdate() 
+    void FixedUpdate() 
     {
         // handoff fixedupdate to current state
         currentState.FixedUpdateState(this);
     }
 
+    // for recording, runs after all update methods are completed
+    private void LateUpdate()
+    {
+        // record replay data for this frame
+        ReplayData data = new PlayerReplayData(this.transform.position, facingRight);
+        recorder.RecordReplayFrame(data);
+    }
+
+    // temporary facing handle script
+    private void HandleFacing()
+    {
+        Vector3 eyePosition = eyes.transform.localPosition;
+        eyePosition.z = 0;
+        eyePosition.y = 0.57f;
+
+        if (facingRight)
+        {
+            eyePosition.x = 0.23f;
+        }
+        else
+        {
+            eyePosition.x = -0.23f;
```

Comment:
Would be a good idea to change these values to fields in the future

Reply:
Hey, sorry I don't know what you mean by this, can you please provide an example?

Reply:
Like make it a [SerializeField] variable up top so we can change it from editor instead of code.